apps/google-analytics-4/frontend/src/components/main-app/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  ChartData,
+  ChartOptions,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
+import tokens from '@contentful/f36-tokens';
+
+const ACCESSIBILITY_LABEL = 'Analytics line chart';
+
+ChartJS.register(CategoryScale, LinearScale, PointElement, LineElement, Title, Tooltip);
+
+ChartJS.defaults.font.size = 12;
+ChartJS.defaults.font.family = tokens.fontStackPrimary;
+ChartJS.defaults.font.weight = '500';
+ChartJS.defaults.borderColor = tokens.gray200;
+ChartJS.defaults.datasets.line.borderColor = tokens.colorPrimary;
+
+interface LineChartProps {
+  dataValues: number[];
+  xAxisLabels: string[];
+  tooltipMetricLabel: string;
+  accessibilityLabel: string;
+}
+
+const LineChart = ({
```

Comment:
nit: Just a note here for convention consistency.
Change `LineChartProps to Props` and then destructure your props below the function definition.

```
const LineChart = (props: Props) => {
  const { dataValues, xAxisLabels, tooltipMetricLabel, accessibilityLabel } = props
  // rest of the code 
}
```

Reply:
Thanks for pointing this out! Have adjusted!